### ISSUE TITLE
Impossible to load library on Mac OS X

### ISSUE THREAD

**Issue by matpascaud** [openpnp/opencv#91]
Hi,

I followed the instruction to use OpenCV on my Mac but impossible to make it work.
I installed OpenCV v4.6.0-0 using maven.
Then I run mvn clean install.

However when trying to load via OpenCV.loadLocally(), OpenCV.loadShared(); or via loadLibrary I am getting the following error :
```
Native code library failed to load.
java.lang.UnsatisfiedLinkError: /tmp/opencv_openpnp2954298317405770839/nu/pattern/opencv/linux/x86_64/libopencv_java460.so: Error loading shared library ld-linux-x86-64.so.2: No such file or directory (needed by /tmp/opencv_openpnp2954298317405770839/nu/pattern/opencv/linux/x86_64/libopencv_java460.so)
```
Did I miss something? Why it's detecting Mac OS as a Linux? I am running on a Mac X86_64.

Thank you for your help.

**Comment by vonnieda** [comment]
Are you still having this issue?

**Comment by vonnieda** [comment]
Please try the latest version and re-open if this is still relevant. Should be fixed in v4.7.0.
